HTAN.model.csv

```diff
@@ -105,28 +105,44 @@ GeoMx DSP NGS Library Preparation Kit Name,Name of Library Preparation Kit. Stri
+ROI name,ROI name (application generated),,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+Segment name,Name given to segment at time of generation,,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+Tags,Unique descriptor of a variable group (ie. MAPK+),,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+ROI X Coordinate,X location within the image,,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+ROI Y Coordinate,Y location within the image,,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+QC status,ROI quality control flag as reported by the application,,,,FALSE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Scan Height,Height of the scan for GeoMx Analysis,,,,TRUE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Scan Width,Width of the scan for GeoMx Analysis,,,,TRUE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Scan Offset X,Offset X of the scan for GeoMx Analysis,,,,TRUE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Scan Offset Y,Offset Y of the scan for GeoMx Analysis,,,,TRUE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Binding Density,The binding density as reported by the application,,,,FALSE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Positive norm factor,The Positive Control Normalization factor calculated using pos-hyb controls,,,,FALSE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+Surface area,Surface area of the ROI in square microns (µm2),,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+Nuclei count,Number of nuclei detected in the segment (if applicable),,,,TRUE,"NanoString GeoMx DSP ROI RCC Segment Annotation Metadata, NanoString GeoMx DSP ROI DCC Segment Annotation Metadata",,,
+Tissue Stain,e.g. CD45 or PanCK (if masking was performed),,,,FALSE,NanoString GeoMx DSP ROI RCC Segment Annotation Metadata,,,
+NanoString GeoMx DSP ROI DCC Segment Annotation Metadata,GeoMx ROI and Segment Metadata Attributes. The assayed biospecimen should be reported one per row with the associated ROI coordinates. ,,"HTAN Parent Biospecimen ID, Scan name, Slide name, ROI name, Segment name, ROI X Coordinate,ROI Y Coordinate, Tags, Scan Height, Scan Width, Scan Offset X, Scan Offset Y, Surface area, Nuclei count, MapQ30, Raw reads, Stitched reads,
+Aligned reads, Deduplicated reads, In Situ Negative median, Biological probe median",,FALSE,Assay,,,

```

Comment:
Not sure if this affects anything, but we may want to fix line spacing in the DependsOn column